doc-content/enterprise-only/springboot/spring-boot-jms-audit-proc.adoc

```diff
@@ -0,0 +1,126 @@
+[id='spring-boot-jms-audit-proc_{context}']
+= Replicating audit data in a JMS message broker
+
+You can replicate {KIE_SERVER} audit data to a Java Message Service (JMS) message broker, for example ActivMQ or Artemis, and then dump the data in an external database schema so that you can improve the performance of your Spring Boot application by deleting the audit data from your application schema.
+
+If you configure your application to replicate data in a message broker, when an event occurs in {KIE_SERVER} the record of that event is stored in the {KIE_SERVER} database schema and it is sent to the message broker. You can then configure an external service to consume the message broker data into an exact replica of the application's database schema. The data is appended in the message broker and the external database every time an event is produce by {KIE_SERVER}.
+
+NOTE: Only audit data is stored in the message broker. No other data is replicated.
+
+.Prerequisites
+* You have a {PRODUCT} Spring Boot application.
+
+.Procedure
+. Open the Spring Boot application's `pom.xml` file in a text editor.
+. Add the {KIE_SERVER} Spring Boot audit dependency to the `pom.xml` file:
++
+[source, xml]
+----
+<dependency>
+  <groupId>org.kie</groupId>
+  <artifactId>kie-server-spring-boot-autoconfiguration-audit-replication</artifactId>
+  <version>${version.org.kie}</version>
+</dependency>
+----
+. Add the dependency for your JMS client. The following example adds the Advanced Message Queuing Protocal (AMQP) dependency:
++
+[source, xml]
+----
+<dependency>
+			<groupId>org.amqphub.spring</groupId>
+			<artifactId>amqp-10-jms-spring-boot-starter</artifactId>
+			<version>2.2.6</version>
+		</dependency>
+----
+. Add the JMS pool dependency:
++
+[source, xml]
+----
+<dependency>
+  <groupId>org.messaginghub</groupId>
+  <artifactId>pooled-jms</artifactId>
+</dependency>
+----
+. To configure {KIE_SERVER} audit replication to use queues, complete the following tasks:
++
+.. Add the following lines to your Spring Boot application's `application.properties` file:
++
+[source]
+----
+kieserver.audit-replication.producer=true
+kieserver.audit-replication.queue=audit-queue
+----
+.. Add the properties required for your message broker client. The following example shows how to configure {KIE_SERVER} for AMPQ where `<JMS_HOST_PORT>` is the port that the broker listens on and  `<USERNAME>` and `<PASSWORD` are the login credentials for the broker:
++
+[source]
+----
+amqphub.amqp10jms.remote-url=<JMS_HOST_PORT>
```

Comment:
I would follow same approach as per the rest of properties in this document and add the prefix `amqp://` to the value:

```suggestion
amqphub.amqp10jms.remote-url=amqp://<JMS_HOST_PORT>
```